src/styles/utilites/_variables.scss

```diff
@@ -0,0 +1,12 @@
+
+$darcBlue: #060b35;
+$backgroundColor: #fff;
+$borderColor: #f3f3f3;
+$grey: #616070;
+$blue: #00acdc;
+$black: #000;
+$buttonBorderDefault: rgba(255, 255, 255, 0);
+$card-width: 200px;
+$space-column: 48px;
+$space-row: 46px;
+$catalog-width: (($card-width) * 4) + ($space-column * 3);
```

Comment:
You have extra parentheses, I think you can easily remove them: $catalog-width: ($card-width * 4) + ($space-column * 3);